crates/cxx-qt-gen/src/naming/rust.rs

```diff
@@ -257,10 +258,10 @@ mod tests {
     #[test]
     fn test_syn_type_cxx_bridge_to_qualified_mapped() {
         let mut mappings = TypeNames::default();
-        mappings.insert("A", Some(parse_quote! { ffi::B }), None, None);
+        mappings.insert("A", Some(format_ident!("ffi")), None, None);
```

Comment:
Even though it's for tests i wonder if it would be nicer to use a new type for these args, as it's not obvious what None, None means. Eg if there was `struct CxxName(Option<&str>)` etc this could instead read.

```rust
mappings.insert("A", QualifiedPath(Some(format_ident!("ffi"))), CxxName(Some("B")), Namespace(None));
```

I would also ask why is `ident` here a `&str` and not a `&Ident` ?

Something like the `Namespace` newtype could be generally useful elsewhere too ?

Reply:
Yup, for the tests it doesn't matter much, so not much of a problem here. Just a thought for any non-test methods that we could have better types in the refactor too.

Reply:
Right, we'll have to see which types benefit from more encapsulation when we get around to rewriting the name generation after we split the parsing...

Probably the Namespace. As that has quite a few constraints on it...
Maybe also a CXX identifier or something :thinking: 
As then a namespace would be basically just a `Vec<CxxIdent>`.
That CxxIdent type could make sure that the CXX type is actually a valid identifier in C++.